charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -101,7 +101,12 @@ data:
 {{ include "prometheus.keep-metrics.metric-relabel-config" .Values.allowedMetrics.kubeETCD3 | indent 6 }}
 
     - job_name: kube-etcd3-backup-restore
-      scheme: http
+      scheme: https
+      tls_config:
+        # Etcd backup sidecar TLS reuses etcd's TLS cert bundle
+        insecure_skip_verify: true
```

Comment:
Why do we do it insecurely here?

Reply:
Because the etcd certs are not generated for a specific pod IP, so client can't really verify server authenticity. This is the case with all TLS scrape configs in prometheus.